### DISCUSSION TITLE
Use system accent color for controls and spectrum as default

### DISCUSSION THREAD

**Discussion by RafaelKr** [wwmm/easyeffects#1790] [Ideas]
First: Coming from PulseEffects I was very surprised how pleasing the new UI for EasyEffects looks! Very nice job, I love it! Also it works perfectly for my use case (Noise cancellation for my microphone input).

What I think would increase the UX even more would be to use colors matching the system for controls (toggles, checkboxes, etc.) and also for the spectrum by default (system accent and text color). I'm not sure if this is possible with the flatpak version, but maybe something to investigate.

As an example I would like to have that mint green from my Linux Mint 21:

![image](https://user-images.githubusercontent.com/14234815/187641261-54a264c3-17a9-4919-9a7f-96cfdca3e346.png)

Edit: Just found the somehow related https://github.com/wwmm/easyeffects/discussions/1475 (Theme wrong)

**Comment by vchernin** [comment]
For the most part the best way to fix this is outside of EasyEffects. Work needs to be done in libadwaita to introduce an API for its existing css recoloring support. Furthermore there needs to be a another key in the settings portal to communicate the preference across apps. Then this will work properly out of the box provided mint supports the portal.

That said in the meanwhile you should be able to manually tweak with libadwaita colors. There is an [interesting new app available](https://flathub.org/apps/details/com.github.GradienceTeam.Gradience) to apparently help set colors. I have not tested it.

**Reply by wwmm** [reply]
I think that in PulseEffects I tried to use the accent color. The problem is that it doesn't feel like gtk wants developers to be doing this. I had to literally look at raw css libadwaita code to find the color names. It was a weird experience. And depending on the accent color a black background is not a good idea. So a decent color for the spectrum background would have to be automatically found. But the theme background doesn't always look good in the spectrum. In the end I decided to just let the user choose one.

**Reply by RafaelKr** [reply]
@vchernin Thanks for that tool suggestion, I tried it without success. Maybe because I use EasyEffects with flatpak?

**Reply by vchernin** [reply]
It would be best to ask them at their support channel, but do also make sure you follow their instructions: https://github.com/GradienceTeam/Gradience/#libadwaita-applications. I imagine you'd need to restart EasyEffects as well.

**Reply by RafaelKr** [reply]
Aaaaah nice! I wasn't aware there is an additional configuration required. Following the install steps from the readme and restarting EasyEffects did the trick, awesome!

**Reply by RafaelKr** [reply]
First: This is how it looks now, it integrates very well with the rest of the system :)
![image](https://user-images.githubusercontent.com/14234815/189142439-5e1eea88-9773-49da-b0cc-35b1b031cdd9.png)

Second: 
With the next release of Gradience there is no additional configuration required anymore, if I understand it correctly
https://github.com/GradienceTeam/Gradience/issues/276
